test/validation/api/ipsec/ipsec_test_out.c

```diff
@@ -431,41 +431,35 @@ static void ipsec_pkt_auth_err_set(odp_packet_t pkt)
 	odp_packet_copy_from_mem(pkt, len - sizeof(data), sizeof(data), &data);
 }
 
+static void ipsec_pkt_update(odp_packet_t pkt, const ipsec_test_flags *flags)
+{
+	if (flags && flags->stats == IPSEC_TEST_STATS_PROTO_ERR)
+		ipsec_pkt_proto_err_set(pkt);
+
+	if (flags && flags->stats == IPSEC_TEST_STATS_AUTH_ERR)
+		ipsec_pkt_auth_err_set(pkt);
+}
+
 static void ipsec_check_out_in_one(const ipsec_test_part *part_outbound,
 				   const ipsec_test_part *part_inbound,
 				   odp_ipsec_sa_t sa,
 				   odp_ipsec_sa_t sa_in,
 				   const ipsec_test_flags *flags)
 {
 	int num_out = part_outbound->num_pkt;
+	ipsec_test_packet pkt_in[num_out];
```

Comment:
Why was this moved here and changed to an array? It is only used as a temporary variable inside the loop.

Reply:
Agreed. Some artifact from the rebase effort. Will have this addressed in v18.